e2e-tests/contentful/cypress.json

```diff
@@ -1,7 +1,7 @@
 {
   "baseUrl": "http://localhost:9000",
   "retries": {
-    "runMode": 2,
+    "runMode": 0,
```

Comment:
Retries when using @cypress/snapshot leads to extra snapshots instead of failed tests. Thats why this PR enables retry for the image related tests, only.

Reply:
This is also why the tests passed instead of failing with the new remark version and changed HTML output!